src/lib/ChapterMap.svelte

```diff
@@ -19,7 +19,10 @@
   <Map
     markers={chapters.map((chap) => ({
       ...chap.coords, // contains { lng, lat }
-      classes: [`chapter`, chap.acceptsSignups ? `active` : `starting`],
+      classes: [
+        `chapter`,
+        chap.acceptsSignups ? `active` : chap.partnerAssociation ? 'partner' : `starting`,
+      ],
```

Comment:
Similar suggestion to #98, how about instead of defining a bunch of booleans, we add an attribute `status: 'active' | 'starting' | 'partner'` or similar to the `Chapter` type?

Reply:
But we keep the bunch of booleans in Contentful right? There is no nice way to implement that you can select from a set of possible states.

Reply:
acceptSignups is the boolean variable that is currently directly loaded from Contentful. Maybe we should add an additional state for the signupsState?

Reply:
> There is no nice way to implement that you can select from a set of possible states.

Yes there is. I just added a "state" to chapters. Take a look on Contentful for how to set that up.

![Screenshot 2023-03-26 at 08 06 05](https://user-images.githubusercontent.com/30958850/227784872-4106550a-cf8f-4e92-8816-e48086307320.png)

Reply:
I second this, this would be more elegant and easier to modify in the future.